test/AutoDiff/validation-test/inout_parameters.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-run-simple-swift(-Xfrontend -requirement-machine=off)
+// RUN: %target-run-simple-swift(-Xfrontend)
```

Comment:
You need to remove the `(-Xfrontend)` here, or the test will fail.

Reply:
Good catch!